dev-lang/ocaml/ocaml-4.09.0.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+HOMEPAGE="https://ocaml.org/"
+SRC_URI="https://github.com/ocaml/ocaml/archive/${PV}.tar.gz -> ${P}.tar.gz"
+DESCRIPTION="Programming language supporting functional, imperative & object-oriented styles"
+
+LICENSE="LGPL-2.1"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+IUSE="emacs flambda latex ncurses spacetime X xemacs"
+
+RDEPEND="sys-libs/binutils-libs:=
+	ncurses? ( sys-libs/ncurses:0= )
```

Comment:
How does it affect the installation? A user will have to reinstall ocaml every time ncurses is toggled, but it seems to be pointless.

Reply:
I forgot to take the ncurses USE flag out before pushing, it did not make sense to me either.